src/dune_rules/dune_package.ml

```diff
@@ -330,6 +330,7 @@ let decode ~lang ~dir =
         [ Pp.textf "The section %s appears multiple times" (to_string s) ]
   in
   let sections =
+    let sections = List.map sections ~f:(fun (k,v) -> (k, Path.of_string v)) in
```

Comment:
`Path.of_string` seems wrong to me. Are you sure you want to accept absolute paths here?

Reply:
In that case, why not require absolute paths directly with `Path.External.of_string`?